src/seedu/addressbook/AddressBook.java

```diff
@@ -625,8 +625,8 @@ private static boolean cleanAndCheckInputLine(String inputLine) {
     /**
      * Shows a message to the user
      */
-    private static void showToUser(String... message) {
-        for (String m : message) {
+    private static void showToUser(String... messages) {
```

Comment:
varargs is not a good way to pass multiple messages. Hint: ArrayList or Array? Separate arguments?